easybuild/easyconfigs/g/GDAL/GDAL-2.2.0-intel-2017a-Python-2.7.13-HDF5-HDF.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.0'
+hdf5_ver = '1.10.1'
+hdf_ver = '4.2.13'
```

Comment:
@jhein32 `hdf5_ver` and `hdf_ver` are only used in one place now, so no need to use these local variables anymore?

Reply:
Only used in one place.  I was thinking the same but preferred it this way.  You feel strongly about it?